template/.eslintrc.js

```diff
@@ -6,30 +6,238 @@ module.exports = {
+    semi: 0, // require or disallow use of semicolons instead of ASI
+    'sort-vars': 0, // sort variables within the same declaration block (off by default)
+    'space-in-brackets': 0, // require or disallow spaces inside brackets (off by default)
+    'space-in-parens': 0, // require or disallow spaces inside parentheses (off by default)
+    'space-infix-ops': 1, // require spaces around operators
+    'space-unary-ops': [1, { words: true, nonwords: false }], // require or disallow spaces before/after unary operators (words on by default, nonwords off by default)
+    'max-nested-callbacks': 0, // specify the maximum depth callbacks can be nested (off by default)
+    'one-var': 0, // allow just one var statement per function (off by default)
+    'wrap-regex': 0, // require regex literals to be wrapped in parentheses (off by default)
+
+    // Legacy
+    // The following rules are included for compatibility with JSHint and JSLint. While the names of the rules may not match up with the JSHint/JSLint counterpart, the functionality is the same.
+
+    'max-depth': 0, // specify the maximum depth that blocks can be nested (off by default)
+    'max-len': 0, // specify the maximum length of a line in your program (off by default)
+    'max-params': 0, // limits the number of parameters that can be used in the function declaration. (off by default)
+    'max-statements': 0, // specify the maximum number of statement allowed in a function (off by default)
+    'no-bitwise': 1, // disallow use of bitwise operators (off by default)
+    'no-plusplus': 0, // disallow use of unary operators, ++ and -- (off by default)
+
+    // React Plugin
+    // The following rules are made available via `eslint-plugin-react`.
+
+    'react/display-name': 0,
+    'react/jsx-boolean-value': 0,
+    'react/jsx-no-comment-textnodes': 2,
+    'react/jsx-no-duplicate-props': 2,
+    'react/jsx-no-undef': 2,
+    'react/jsx-sort-props': 0,
+    'react/jsx-uses-react': 1,
+    'react/jsx-uses-vars': 1,
+    'react/no-did-mount-set-state': 1,
+    'react/no-did-update-set-state': 1,
+    'react/no-multi-comp': 0,
+    'react/no-string-refs': 1,
+    'react/no-unknown-property': 0,
+    'react/no-unstable-nested-components': 1,
+    'react/prop-types': 0,
+    'react/react-in-jsx-scope': 1,
+    'react/self-closing-comp': 1,
+    'react/wrap-multilines': 0,
+
+    // React Hooks Plugin
+    // The following rules are made available via `react-hooks`.
+
+    'react-hooks/exhaustive-deps': 'off',
+    'react-hooks/rules-of-hooks': 'error',
+
+    // React-Native Plugin
+    // The following rules are made available via `eslint-plugin-react-native`
+
+    'react-native/no-inline-styles': 1,
+
+    // Jest Plugin
+    // The following rules are made available via `eslint-plugin-jest`.
+    'jest/no-disabled-tests': 1,
+    'jest/no-focused-tests': 1,
+    'jest/no-identical-title': 1,
+    'jest/valid-expect': 1,
```

Comment:
Are all those rules that you have a strong opinion that we should change? 🤔

Reply:
It's copied rules, as I remember)